### PR TITLE
[office-js-preview] Pulling latest type definitions from CDN

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://appsforoffice.officeapps.live.com/afo/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -76488,6 +76707,10 @@ declare namespace Word {
      * The insertion location types.
      *
      * [Api set: WordApi]
+     * @remarks
+     * To be used with an API call, such as `obj.insertSomething(newStuff, location);`
+     * If the location is ""Before"" or ""After"", the new content will be outside of the modified object.
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:76712]:
```suggestion
     * If the location is "Before" or "After", the new content will be outside of the modified object.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -76488,6 +76707,10 @@ declare namespace Word {
...
+     * If the location is ""Start"" or ""End"", the new content will be included as part of the modified object.
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:76713]:
```suggestion
     * If the location is "Start" or "End", the new content will be included as part of the modified object.
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -42903,10 +43035,15 @@ declare namespace Excel {
          */
         linkedDataTypeLinkedDataTypeAdded = "LinkedDataTypeLinkedDataTypeAdded",
         /**
-         * WorksheetFormulaChanged represents the type of event registered on a worksheet, and occurs when a formula is changed.
+         * WorksheetFormulaChanged represents the type of event registered on worksheet, and occurs when formula is changed.
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:43038]:
```suggestion
         * WorksheetFormulaChanged represents the type of event registered on a worksheet, and occurs when a formula is changed.
```

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -42903,10 +43035,15 @@ declare namespace Excel {
...
+         *
+         */
+        worksheetFormulaChanged = "WorksheetFormulaChanged",
+        /**
+         * WorkbookActivated represents the type of event which is fired when workbook is activated.
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:43043]:
```suggestion
         * WorkbookActivated represents the type of event which is fired when a workbook is activated.
```

**Review by alison-mk — APPROVED** [review]
Approving with suggestions. (Note: my suggestion for WorkbookActivated is new and I have documented the request on our spreadsheet.) 
@AlexJerabek can you review the WordApi additions?

**Review by AlexJerabek — APPROVED** [review]
Looks good @Rick-Kirkham. Let's make sure those Word remarks properly build when we regenerate the reference docs.

**Pull request merged**